app/src/main/java/com/tictactoe_master/data_to_file/DataToFile.kt

```diff
@@ -0,0 +1,84 @@
+package com.tictactoe_master.data_to_file
+
+import android.content.Context.MODE_PRIVATE
+import android.content.SharedPreferences
+import androidx.appcompat.app.AppCompatActivity
+import com.tictactoe_master.R
+
+/*
+*   Before passing or reading values from file first
+*   use method setContext(AppCompatActivity).
+*/
+object DataToFile {
```

Comment:
1. Class name should be more general eg. DataHandler or there shoudl be an interface DataHandler and a derived class FileDataHandler
2. It's reduntant to create a `data_to_file` package for just one file with basically the same name: `DataToFile`: either delete the package or refactor it to include an interface